src/main/java/visualizations/LineChart.java

```diff
@@ -14,6 +16,23 @@
 
 public class LineChart {
 
+    public class LinePlotOptions
```

Comment:
ok this class is public. but what about the parameters. should we make them private and have some getters & setters? is it getting more verbose that way? perhaps use Lombok?

Reply:
I would say leave it public...what is Lombok?

Reply:
I am not sure that we should leave it public bu ok for now. [Lombok](https://projectlombok.org) nice one! helps with over typing check it out.

Reply:
In this case we just want to take some variables from one place to another that's all. This class is not meant to do smart things just moving data from one place to another. So I think it's ok having it like this

Reply:
agreed. will refactor the other classes too on my branch.